readthedocs/core/models.py

```diff
@@ -27,10 +28,6 @@ def __unicode__(self):
         return (ugettext("%(username)s's profile")
                 % {'username': self.user.username})
 
-    def get_form(self):
```

Comment:
Aye. I grepped the *.py files, not the html also, silly me. Should be fixed now.